### PR TITLE
samples: refactored samples to v2

### PR DESCRIPTION
**Pull request by munkhuushmgl**
Fixes the problem with location_path

#4663

### CODE REVIEW

File: iot/api-client/manager/manager.py [hunk 1]
```diff
@@ -93,6 +94,12 @@ def create_rs256_device(
 ):
     """Create a new device with the given id, using RS256 for
     authentication."""
+    # [START iot_create_rsa_device]
```
Comment (busunkim96) [inline iot/api-client/manager/manager.py:97]: Please remove the region tags from this file. This file is only being used for test setup. The version displayed to users is in python-iot.

Comment (munkhuushmgl) [reply iot/api-client/manager/manager.py:97]: ok

**Review by engelke — APPROVED** [review]
LGTM

**Comment by munkhuushmgl** [comment]
@busunkim96 friendly ping

**Review by busunkim96 — APPROVED** [review]

**Pull request merged**